### ISSUE TITLE
Pow! Error no such tables

### ISSUE THREAD

**Issue by Totallytorsten** [InstaPy/InstaPy#2611]
## Current Behavior

Program opens for about a second and then closes. Error is as follows:

> Pow! Error occurred while dumping follow restriction data to a local JSON: b'no such table: followrestriction'
> Pow! Error occurred while dumping follow restriction data to a local JSON: b'no such table: recordActivity'

The program then crashes because it cant find recordActivity. Downloaded a totally fresh version and getting the same error. Think it has to do with the changes to database_engine from earlier this morning.

**Comment by uluQulu** [comment]
@Totallytorsten,
Are your files up-to-date?   (_see `git status`_)

If you got the latest files and the problem still persists, let me know. I will help.

-------------

Cheers 😁

**Comment by eddyizm** [comment]
Getting the same error. git status shows everything already up to date.

**Comment by uluQulu** [comment]
I have checked, @eddyizm.
You are right, @Totallytorsten, it is a bug born off #2605 🐝

Wait a few minutes, I will share a solution to it and explain the problem details 🤝🏼

**Comment by uluQulu** [comment]
Hey @eddyizm, @Totallytorsten
The fix is ready - #2627 ☄

**Comment by eddyizm** [comment]
Thanks @uluQulu . I actually ended up renaming the db and let it run again, thus generating a new db and everything was well. Should I go back to the old db and let the app handle the upgrade or can I leave it alone, not sure how much I'm giving up with all the old stats?

**Comment by uluQulu** [comment]
Smart of you @eddyizm 🧐

Well, it does not matter, cos as of now, the data stored in the `statistics` (_now replaced by `recordActivity` table_) table is not in use by any module, it is just a useful data to see but in future it will be crucial and will be used by internal modules.
Anyways, even if you let the program handle the upgrade, it will make a new table `recordActivity` for that reason, and the `statistics` table will not be used.  

The reason I designed it so is I thought there are some people who uses the data inside `statistics` table and I kept it for them to take backups.  

------
Cheers 😁
